release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1474,8 +1474,16 @@ In the following tables, features are marked with the following statuses:
 |General Availability
 |Removed
 |Removed
-|====
 
+|Shared Resources CSI Driver in OpenShift Builds
+|Deprecated
+|Deprecated
+|Deprecated ^[1]^
```

Comment:
This should not be deprecated In OpenShift Builds, right?, It should be deprecated in OCP